### PR TITLE
Add adapter for IAS

### PR DESCRIPTION
**Pull request by katzhang-ias**
## Type of change

- [x] New bidder adapter   

## Description of change

This module is an integration with prebid.js with an IAS product, pet.js. It is not a bidder per se but works in a similar way: retrieve data that publishers might be interested in setting keyword targeting.

- test parameters for validating bids
```
                   {
                       bidder: "ias",
                       params: {
                           pubId: '99',
                           adUnitPath: '/57514611/news.com'
                       }
                   }
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: kat@integralads.com
- [x] official adapter submission

### CODE REVIEW

File: modules/iasBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+function interpretResponse(serverResponse, request) {
+  const iasResponse = serverResponse.body;
+  const bidResponses = [];
+
+  // Keys in common bid response are not used;
+  // Necessary to get around with prebid's common bid response check
+  const commonBidResponse = {
+    requestId: request.bidRequest.bidId,
+    cpm: 1,
```
Comment (matthewlane) [inline modules/iasBidAdapter.js]: Can this be changed to `0.01`? The hardcoded `1` may prevent bidders with real cpms from winning

**Comment by katzhang-ias** [comment]
Hi @matthewlane , 
I changed the cpm value per review. Please take a look when you have a chance. 
Thanks!

**Comment by matthewlane** [comment]
@katzhang-ias Please also submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html)

**Comment by katzhang-ias** [comment]
Thanks @matthewlane ! Will do.

**Comment by katzhang-ias** [comment]
Docs PR: https://github.com/prebid/prebid.github.io/pull/595

**Comment by katzhang-ias** [comment]
Please let me know if there's anything else that needs to be done to get it merged/released.

**Pull request merged**